keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -229,20 +306,37 @@ https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05
 
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
+If strict decoding fails, proxy fall back to a lenient decoder, ensuring  that
+rollouts and rollbacks remain unaffected in case of a malformed configuration.
+
+Running workloads are not affected.
+
 ###### What specific metrics should inform a rollback?
 
+N/A
```

Comment:
Shouldn't we point at e.g. synx_proxy_rules_duration_seconds being empty or fairly high (suggesting some misconfiguration)?